src/main/java/CommandHandler.java

```diff
@@ -0,0 +1,142 @@
+import java.util.Scanner;
+
+public class CommandHandler {
+    private static Task[] tasks = new Task[100];
+    private static int taskCounter = 0;
+    public static void drawLine() {    //print underscore symbol 50 times
+        System.out.println("__________________________________________________");
+    }
+    public static void byeUser() {
+        System.out.println("Bye. Hope to see you again!");
+        drawLine();
+    }
+    public static void listTasks() {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < taskCounter; ++i) {
+            char taskTypeCharacter = tasks[i].getType();
+            System.out.println(i+1 + ".[" + taskTypeCharacter + "]" +
+                    "[" + (tasks[i].isDone() ? "X] " : " ] ")
```

Comment:
perhaps the use of the lambda function here makes the code slightly less readable